docs/containerd-env-setup.md

```diff
@@ -30,7 +30,7 @@ $ sudo tee /etc/nydusd-config.json > /dev/null << EOF
       "type": "registry",
       "config": {
         "scheme": "",
-        "skip_verify": false,
+        "skip_verify": true,
         "timeout": 5,
         "connect_timeout": 5,
         "retry_limit": 2,
```

Comment:
From our test results, the value 2 is too low. I suggest to enlarge it to at least 4 or 8